docs/admin.md

```diff
@@ -0,0 +1,59 @@
+## Admin Manual for Definitely Typed
+
+Welcome! This is a resource for the person who is on call for DefinitelyTyped. The TypeScript team always has someone 
+assigned to DefinitelyTyped duty, where they do a week on-call. You can see the [schedule here](http://aka.ms/DTRotation).
+
+When on-call, your goal is to try keep on top of the many open PRs for DefinitelyTyped, they are categorized into 
+different sections inside the [Projects board](https://github.com/DefinitelyTyped/DefinitelyTyped/projects/4) on this repo.
+
+You should scan from left to right through the boards, and ideally try to start at the latest PR and work your way 
+through to the newest.
+
+There is a tool: [`focus-dt`](https://github.com/DefinitelyTyped/focus-dt) which can help with this.
+
+### Looking at a PR
+
+Some useful heuristics for looking at a PR:
+
+##### New DefinitelyTyped Package
+
+- Is the author also a maintainer of the library? If so, they could use [`--declaration` and `--allowJs`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#--declaration-and---allowjs) in TypeScript to generate their types.
+
```

Comment:
at some point we should have the content of 'oh no modules' in here, plus links to the handbook. for now can you add a list like

* no exceptions in tslint.json
* `strict: true` or equivalent four (five?) settings in tsconfig.json
* no `esModuleInterop: true`, no matter how tempting
* make sure export default is actually default in the source; react-* packages get a pass on this.

Also maybe link to my slide deck: https://docs.google.com/presentation/d/1Q4xfZSY7d9yHhtxSyb-DE85fTXB38RyF3nnyVyvenwc/edit?usp=sharing